package.json

```
@@ -10,35 +10,36 @@
     "start": "node lib/index.js",
     "test": "grunt test"
   },
-  "repository": "https://github.com/tidepool-org/message-api.git",
+  "repository": "https://github.com/mdblp/message-api",
   "author": {
     "name": "Jamie Bate"
   },
   "license": "BSD-2-Clause",
   "dependencies": {
-    "amoeba": "0.2.1",
-    "bunyan": "1.4.0",
-    "cookies": "0.5.0",
-    "hakken": "0.1.8",
-    "lodash": "3.4.0",
-    "mongojs": "2.4.0",
-    "request": "2.53.0",
-    "restify": "2.8.5",
-    "sundial": "1.4.0",
-    "tidepool-gatekeeper": "0.2.2",
-    "tidepool-seagull-client": "0.1.5",
-    "user-api-client": "0.4.0"
+    "amoeba": "0.4.3",
+    "bunyan": "1.8.12",
+    "cookies": "0.7.3",
+    "hakken": "0.1.12",
+    "lodash": "4.17.11",
+    "moment": "^2.24.0",
+    "mongojs": "2.6.0",
+    "request": "2.88.0",
+    "restify": "8.3.2",
+    "sundial": "1.6.0",
+    "tidepool-gatekeeper": "0.2.4",
+    "tidepool-seagull-client": "0.1.8",
+    "user-api-client": "mdblp/user-api-client#feature/pt583-v0.5.0"
```

We don't use our forked version. 
```suggestion
    "user-api-client": "0.5.0"
```